tests/bootstrap.php

```diff
@@ -0,0 +1,2 @@
+<?php
+@include_once dirname(__FILE__) . '/../vendor/autoload.php';
```

Comment:
@singpolyma thank you for the feedback. Correct, the `vendor/autoload.php` is generated by composer and it sets up [the PHP class autoloader](http://php.net/manual/en/language.oop5.autoload.php) and the include path (in this case for phpseclib).

Without composer the `include_once` would not do anything and with the `@` in front of it, there would be no warnings about the missing file (I could also do a file_exists first). So it doesn't add a dependency on composer - it is just included if it's there. 

But I admit the bootstrap thing isn't really required for people who want to just load openpgp-php as a dependency to their project - they just need the composer.json. The bootstrap-change would just make things easier for developers who actually work on openpgp-php itself and are using composer to resolve the phpseclib-dependency.

So if you think that this change is too much - which I can fully understand - you could copy the composer.json into the project root and close this PR (or I can do a new PR with just the composer.json in it).

If I was able to convince you that this doesn't force a composer dependency and you are ok with the bootstrap-thing, I will of course add a blank line at the end (sorry for missing this).